VideoLocker/res/layout/fragment_course_dashboard.xml

```diff
@@ -15,30 +16,48 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:scaleType="fitXY"
-            tools:src="@drawable/edx_map_login"/>
+            tools:src="@drawable/edx_map_login" />
 
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="70dp"
             android:layout_gravity="bottom"
             android:background="@color/transparent_white_85"
-            android:orientation="vertical"
+            android:orientation="horizontal"
             android:padding="10dp">
 
-            <org.edx.mobile.view.custom.ETextView
-                android:id="@+id/course_detail_name"
-                style="@style/course_card_title"
-                tools:text="Course Name" />
+            <LinearLayout
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_gravity="bottom"
+                android:layout_weight="15"
+                android:orientation="vertical">
 
-            <org.edx.mobile.view.custom.ETextView
-                android:id="@+id/course_detail_extras"
-                style="@style/course_card_course_run"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                tools:text="XX | xx | xxxxx" />
+                <org.edx.mobile.view.custom.ETextView
+                    android:id="@+id/course_detail_name"
+                    style="@style/course_card_title"
+                    tools:text="Course Name" />
 
+                <org.edx.mobile.view.custom.ETextView
+                    android:id="@+id/course_detail_extras"
+                    style="@style/course_card_course_run"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    tools:text="XX | xx | xxxxx" />
+            </LinearLayout>
+
+            <org.edx.mobile.third_party.iconify.IconButton
+                android:id="@+id/course_detail_share"
+                android:layout_width="0dp"
```

Comment:
I would just use 20dp here and remove layout_weight since this view doesn't need to expand. Also, 20dp is a pretty small target; I suggest removing the `10dp` padding from the parent layout and setting it on each child, so that the effective touch target for this button is 40dp